src/components/molecules/UserProfilePicture/UserProfilePicture.scss

```diff
@@ -1,83 +1,152 @@
 @import "scss/constants.scss";
 
 $fadedWhite: rgba(255, 255, 255, 0.8);
-$borderWidth: 5px;
 
-.profile-picture-container {
-  position: relative;
+// TODO: this was copied from Audience.scss (do we want to move the variable to constants?)
+$avatarSize: max(4vh, 3vw);
+
+// TODO: should we move these to using CSS --variables rather than these SCSS vars
+$reactionContainerOffset: calc(-1 * #{$avatarSize});
+$reactionSize: calc(#{$avatarSize / 1.5});
+$reactionOffset: calc(-1 * #{$reactionSize} / 2);
+$shoutBounceTranslateXOffset: calc(#{$avatarSize} / 2);
+
+.UserProfilePicture {
+  width: 100%;
+  height: 100%;
+
   background-position: center;
   background-size: cover;
 
-  .profile-icon {
-    width: 44px;
-    height: 44px;
-    background-size: 100%;
+  &--reaction-left {
+    .UserProfilePicture__reaction {
+      left: $reactionOffset;
+    }
+
+    .UserProfilePicture__shout {
+      right: 0;
+      transform-origin: right center;
+      animation: shout-bounce--left 5s ease;
+    }
+  }
+
+  &--reaction-right {
+    .UserProfilePicture__reaction {
+      right: $reactionOffset;
+    }
+
+    .UserProfilePicture__shout {
+      left: 0;
+      transform-origin: calc(#{$avatarSize} / 2) center;
+      animation: shout-bounce--right 5s ease;
+    }
+  }
+
+  &__avatar {
+    width: $avatarSize;
+    height: $avatarSize;
+
+    background-color: transparent;
+    // TODO: how can we use a default image here if none set? Can we use CSS --variables here? I think we could..
+    //background-image: url(${({ backgroundImage }) => backgroundImage ?? DEFAULT_PROFILE_IMAGE});
+    background-position: center;
+    background-size: cover;
     background-repeat: no-repeat;
+
+    border-radius: 10rem;
+  }
+
+  &__reaction-container {
+    position: relative;
+
+    top: $reactionContainerOffset;
+    //z-index: -1;
+    //top: 0;
+    //left: 0;
+    width: $avatarSize;
+    height: $avatarSize;
+    //width: 100%;
+    //height: 100%;
+
+    // TODO: remove this
+    background-color: rgba(255, 0, 0, 0.2);
+    border: 1px solid #f00;
+  }
+
+  &__reaction {
+    position: absolute;
+    top: $reactionOffset;
+
+    // Note: emoji reactions are sized with font-size
+    font-size: $reactionSize;
+
+    animation: reaction-pulse 4s ease-in-out infinite;
   }
 
-  .messageToBand {
+  &__shout {
+    position: absolute;
+    bottom: 0;
+
     width: max-content;
     max-width: 20em;
     padding: 6px 10px;
 
-    position: absolute;
-    top: 0;
-    left: 0;
-
-    background: $fadedWhite;
+    // TODO: this is how the old styled-components had it
+    //background-color: rgba(255, 255, 255, 1);
+    // TODO: this is how the old (potentially unused) scss had it
+    background-color: $fadedWhite;
```

Comment:
[TODO] Pick what colour we want these messages to be

@mcflyok Do you have input on this? Which is the correct/better choice here?

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1323/commits/f8b0fcbaf3ffd67f4a1226baea574719e9b22d65